e2e/video-player.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { test, expect } from '@playwright/test';
+
+test.beforeEach(async ({ page }) => {
+  await page.goto(
+    '/learn/scientific-computing-with-python/python-for-everybody/introduction-why-program'
+  );
+});
+
+test.afterEach(async ({ page }) => {
+  await page.close();
+});
+
+test.describe('Challenge Video Player Component Tests', () => {
+  test('should render video player and play button', async ({ page }) => {
+    await expect(
+      page.locator('iframe[title="YouTube video player"]')
```

Comment:
Should we add a playright test attribute to the iframe? I think that's more efficient than a `locator` query.

Reply:
As we are using `YouTube` component from `react-youtube` package, we don't have access to the `iframe`. One other option is to use `locator('_react=YouTube')`.
[Code](https://github.com/freeCodeCamp/freeCodeCamp/blob/225a0fbd059781fa262148e6a21e8f27d6303638/client/src/templates/Challenges/components/video-player.tsx#L2)